requirements.txt

```diff
@@ -372,7 +372,7 @@ snuggs==1.4.7
     # via
     #   rasterio
     #   stdl-object-detector
-stdl-object-detector @ git+https://github.com/swiss-territorial-data-lab/object-detector.git@v2.3.2
+stdl-object-detector @ git+https://github.com/swiss-territorial-data-lab/object-detector.git@v.2.3.3
```

Comment:
How do you perform `pip-compile` with the version tag? For me, it crashes the process.

Reply:
then on my side it was probably crashing because of the extra dot and v2.3.2 must be the correct way to do it.

Reply:
I'm a bit confused by your previous comment. Should we switch to v2.3.2, without dot after v? If yes then we have to change it in the repos affected and in the OD tags.